### PR TITLE
Return URL in startIfStopped Method

### PR DESCRIPTION
**Pull request by williamsyang-work**
Changes needed for a working vscode remote ssh fix.

Signed-off-by: Will Yang <william.yang@ericsson.com>

### CODE REVIEW

**Review by marco-miller — CHANGES_REQUESTED** [review]
LGTM, there is only a missing space in the commit message `Signed-off-by`.

**Comment by williamsyang-work** [comment]
> LGTM, there is only a missing space in the commit message `Signed-off-by`.

Whops!  Fixed.

**Review by marco-miller — APPROVED** [review]
LGTM, yet letting the 2nd review unfold.

File: src/trace-server.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export class TraceServer {
             await context?.workspaceState.update(key, none);
             await this.start(context);
         }
+        return this.getUrl(this.getSettings());
```
Comment (bhufmann) [inline src/trace-server.ts:89]: Shouldn't it be `this.getClientSettings()`? The URL preference is for the client to know where to connect to. 

So, if we return here the client URL, why does the client (i.e. `vscode-trace-extension`) read from the preference directly?

Or do I misunderstand something?

**Review by bhufmann — CHANGES_REQUESTED** [review]
See comment

File: src/trace-server.ts (hunk 1 above)
Comment (williamsyang-work) [reply src/trace-server.ts:89]: Per our discussion we do not need to return the url.  Changes have been made to the other PR, and this change is no longer needed.

**Comment by williamsyang-work** [comment]
Closing this as it's not needed anymore.

**Pull request closed without merging**